common/app/implicits/Requests.scala

```diff
@@ -24,7 +24,7 @@ trait Requests {
 
     lazy val isAmp: Boolean = r.getQueryString("amp").isDefined || (!r.host.isEmpty && r.host == Configuration.amp.host)
 
-    lazy val isEmail: Boolean = (r.getQueryString("format") == Some("email")) || r.path.endsWith(EMAIL_SUFFIX)
+    lazy val isEmail: Boolean = r.getQueryString("format").contains("email") || r.path.endsWith(EMAIL_SUFFIX)
```

Comment:
Actually @johnduffell only `format=email` is allowed through Fastly anyway: https://github.com/guardian/fastly-edge-cache/pull/673